CHANGES.md

```diff
@@ -1,5 +1,16 @@
 ## unreleased (v2.0.0)
 
+### breaking API changes
+
+- The fitters no longer take the observation in the constructor.
+  The `go` method now takes the observation and possibly a guess.
```

Comment:
Did we want to enable the fitters to take a list of mbobs?

this is what is in the mdet repo: https://github.com/esheldon/metadetect/blob/master/metadetect/fitting.py#L43

Reply:
I think we only want the fitters to take single observations.

There are other codes that definitely need to take an mbobs list such as the shredder code.

But for processing different observations, I had in mind that we could do this with the new api
```python
boot = Bootstrapper(...)  # takes runners
reslist = []
for mbobs in mbobs_list:
    boot.go(mbobs)
    reslist.append(boot.fitter.get_result())
```
and then have a code to pack this list of results into an array of structs.  This is pretty compact

Reply:
basically the above code would be used in that metadetect class

Reply:
This sounds great!